src/generated/types/Tree.ts

```diff
@@ -0,0 +1,29 @@
+import { Type } from "@sinclair/typebox";
+import { Category } from "../../constants/category";
+import { StringUnion } from "../../sunmao-helper";
+
+export const TreeNodeSchema = Type.Object({
+  title: Type.String(),
+  key: Type.String(),
+  children: Type.Optional(Type.Array(Type.Any())),
+  selectable: Type.Optional(Type.Boolean()),
+  checkable: Type.Optional(Type.Boolean()),
+});
+
+export const TreePropsSchema = Type.Object({
+  data: Type.Array(TreeNodeSchema, {
+    category: Category.Data,
+    title: "Tree Data",
+  }),
+  size: StringUnion(["mini", "small", "medium", "large"], {
+    category: Category.Style,
```

Comment:
If we don't specify a spec title, the property title will fallback to `schema.name` and the first letter won't be capitalized, should we handle `schema.name` in the editor?

Reply:
It is a mistake. I have added `title` now. `title` is not required, so it is easy to neglect it.